packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -846,6 +846,51 @@ export class BidiPage extends Page {
     });
   }
 
+  override async cookies(): Promise<Protocol.Network.Cookie[]> {
+    const bidiCookies = await this.#connection.send('storage.getCookies', {
+      partition: this.mainFrame()._id,
+    });
+    return bidiCookies.result.cookies.map(c => {
+      return {
+        name: c.name,
+        // Present binary value as base64 string.
+        value: c.value.value,
+        domain: c.domain,
+        path: c.path,
+        size: c.size,
+        httpOnly: c.httpOnly,
+        secure: c.secure,
+        sameSite: this.#convertCookiesSameSite(c.sameSite),
+        // TODO: check default `expiry` value.
+        expires: c.expiry ?? -1,
+        // TODO: add proper value.
+        session: c.expiry === undefined || c.expiry <= 0,
+        // TODO: add proper value.
+        priority: 'Medium',
+        // TODO: add proper value.
+        sameParty: false,
+        // TODO: add proper value.
+        sourceScheme: 'NonSecure',
+        // TODO: add proper value.
+        sourcePort: -1,
```

Comment:
There is no `sourcePort` in BiDi cookie.
What is the expected behavior in case of BiDi?